docs/references/backend/user/update-user-profile-image.mdx

```diff
@@ -18,7 +18,7 @@ function updateUserProfileImage(userId: string, params: { file: Blob | File }):
 
 ```tsx
 const userId = 'user_123'
-const file = new File(['profile-pic'], 'profile-pic.png', { type: 'image/png' })
+const file = new File(['profile-pic-file-contents'], 'profile-pic.png', { type: 'image/png' })
```

Comment:
I think what we have here is fine. Not a strong option but a suggestion 👇🏻 

We could follow what Mozilla shows regarding syntax: 

![CleanShot 2024-09-27 at 10 19 17](https://github.com/user-attachments/assets/87be1c2d-285a-4b6d-af2a-9ea21fe75728)

Extracting to a separate variable:
```suggestion
const fileBits = ["profile-pic-content"]
const fileName = "profile-pic.png"
const file = new File(fileBits, fileName, { type: 'image/png' })
```

Reply:
Even better 😄 . Thank you!